<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Richelieu</class>
 <widget class="QWidget" name="Richelieu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ð¨Ð¸Ñ„Ñ€ Ð Ð¸ÑˆÐµÐ»ÑŒÑ‘</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="MessageLabel">
     <property name="text">
      <string>Ð¡Ð¾Ð¾Ð±Ñ‰ÐµÐ½Ð¸Ðµ</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPlainTextEdit" name="MessageEdit">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="EncodeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ “</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPlainTextEdit" name="CodeEdit">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="CodeLabel">
     <property name="text">
      <string>ÐšÑ€Ð¸Ð¿Ñ‚Ð¾Ð³Ñ€Ð°Ð¼Ð¼Ð°</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="KeyEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="DecodeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ ‘</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="KeyLabel">
     <property name="text">
      <string>ÐšÐ»ÑŽÑ‡</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
